Fagoroye Peter/composition.ts

```diff
@@ -0,0 +1,147 @@
+// COMPOSITION
+
+class Laptop{
+    keyboard: Keyboard[] = []
+    nic: NetworkInterfaceCard
+    screen: Display
+    hardDisk: HardDisk
+    powerState: boolean = false;
+    bitKind: BitKind
+    os: OperatingSystem
+
+    constructor(
+        bit: BitKind,
+        display: Display,
+        nic: NetworkInterfaceCard,
+        hd: HardDisk,
+        os: OperatingSystem,
+        keyboards: Keyboard[]
+    ) {
+        this.powerState = false;
+        this.bitKind = bit;
+        this.screen = display;
+        this.nic = nic;
+        this.hardDisk = hd;
+        this.os = os;
+        this.keyboard = keyboards;
+    }
+
+    switchOn(){
+        this.powerState = true;
+        return "Turning on the Laptop...."
+    }
+
+    switchOff(){
+        this.powerState = false;
+        return "Turning off the Laptop...."
+    }
+
+    update(){
```

Comment:
This is not the assingment.
Meet 'Elochuwkwu' for explanation

Reply:
when i push the commit, we haven't been assigned the shopping cart yet, I would do that now

Reply:
can i still add the shopping cart since you have closed the pull request
or i would have to open another one sir